### ISSUE TITLE
store.epicgames.com: breakage

### ISSUE THREAD

**Issue by muchqs** [uBlockOrigin/uAssets#20421]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] This is NOT a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/19976), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://store.epicgames.com/en-US/p/turnip-boy-commits-tax-evasion-fb2084`

### Category

breakage

### Description

I try to redeem the weekly free game but it constantly gets stuck at "loading order". I think it tries to load from hcaptcha but it's blocked by uBO.

Stuck at loading order:
https://i.imgur.com/Pwotels.png

After I clicked the allow @@ and refreshed, the order dialog loaded up fine quickly.
https://i.imgur.com/BGKxGCs.jpg

Kinda hard to verify I think as I don't get this problem EVERY time. But more often than not this happens when I try to get a game.

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.52.2
Chromium: 116
filterset (summary):
 network: 128722
 cosmetic: 136092
 scriptlet: 26930
 html: 0
listset (total-discarded, last-updated):
 added:
  https://raw.githubusercontent.com/DandelionSprout/adfilt/master/LegitimateURLShortener.txt: 2474-0, 2m
  CHN-0: 25460-135, 2m
  adguard-cookies: 27982-31, 2m
  adguard-mobile-app-banners: 4721-12, 2m
  adguard-other-annoyances: 13723-10, 2m
  adguard-popup-overlays: 25012-66, 2m
  adguard-social: 21970-923, 2m
  adguard-widgets: 2335-3, 2m
  block-lan: 54-0, 2m
  ublock-annoyances: 5681-152, 2m
  ublock-cookies-adguard: 430-430, 2m
 default:
  user-filters: 25-0, never
  easylist: 72285-84, 2m
  easyprivacy: 32852-144, 2m
  plowe-0: 3721-1075, 2m
  ublock-badware: 7421-138, 2m
  ublock-filters: 35129-508, 2m
  ublock-privacy: 641-52, 2m
  ublock-quick-fixes: 229-139, 2m
  ublock-unbreak: 2133-35, 2m
  urlhaus-1: 11555-0, 2m
filterset (user): [array of 25 redacted]
switchRuleset:
 added: [array of 1 redacted]
hostRuleset:
 added: [array of 3 redacted]
userSettings:
 advancedUserEnabled: true
 suspendUntilListsAreLoaded: true
hiddenSettings:
 userResourcesLocation: [redacted]
supportStats:
 allReadyAfter: 4849 ms (selfie)
 maxAssetCacheWait: 318 ms
popupPanel:
 blocked: 0
 extended:
  ##div[class^="css-"] > div[class]:has(> button > span:contains(S…
```
</details>

**Comment by stephenhawk8054** [comment]
So, does this filter work for you?

```adb
@@||epicgames.com^$permissions=browsing-topics=()
```

**Comment by u-RraaLL** [comment]
I tested on Chrome and I wasn't stuck at "loading order" at all (tried about 15 games). OP is not using default settings though.

> Kinda hard to verify I think as I don't get this problem EVERY time.
> After I clicked the allow @@ and refreshed, the order dialog loaded up fine quickly.

Possibly just a refresh would have worked.

**Comment by stephenhawk8054** [comment]
When it happens again, please reset to default settings and test again first, don't add anything.

**Comment by stephenhawk8054** [comment]
@muchqs Can you test this filter?

```adb
||tracking.epicgames.com^$image,redirect-rule=1x1.gif
```

**Comment by u-RraaLL** [comment]
> there is now

Looks to be 5 years old though.

**Comment by muchqs** [comment]
This problem still happens every so often.

The unbreak rule in ublock doesn't help.
Refresh doesn't help.

For the last 4 weeks, it happened every single time when I tried to get the weekly free game.
it just says
"An Error occurred please try again".
The logger didn't show any blocked red entries regarding the domain tracking.epicgames.com

But the only way I can get to the checkout page is to whitelist it with
@@||tracking.epicgames.com/

Then the checkout page loads instantly.

just tried getting this week's free game
https://store.epicgames.com/en-US/p/fear-the-spotlight-97656f
And it happened again.
Had to whitelist the domain to get to checkout or it just errored out nonstop.

**Comment by krystian3w** [comment]
Check again that user topics tracking is unlocked and that the image tracker has loaded:


```adb
@@||epicgames.com^$permissions=browsing-topics=()
||tracking.epicgames.com^$image,important
||tracking.epicgames.com^$image,redirect-rule=1x1.gif:100
```

**Comment by stephenhawk8054** [comment]
Can you do these steps (follow strictly the step order, don't mix the steps orders, or skip any steps):

1. Click on uBO icon > ▤ ["The logger" icon](https://github.com/gorhill/uBlock/wiki/The-logger), it will open a logger window
2. Open new tab and reproduce the issue
3. Switch to the logger window that was opened in step 1, [hover over the search bar, choose `blocked`](https://github.com/gorhill/uBlock/wiki/The-logger#filtering-the-logger-output)
4. Copy the logger via 📋 [export/copy button at the top right](https://github.com/gorhill/uBlock/wiki/The-logger#export-dialog)
5. Paste the logs to https://logpasta.com and share the link here

**Comment by muchqs** [comment]
> Can you do these steps (follow strictly the step order, don't mix the steps orders, or skip any steps):
> 
> 1. Click on uBO icon > ▤ ["The logger" icon](https://github.com/gorhill/uBlock/wiki/The-logger), it will open a logger window
> 2. Open new tab and reproduce the issue
> 3. Switch to the logger window that was opened in step 1, [hover over the search bar, choose `blocked`](https://github.com/gorhill/uBlock/wiki/The-logger#filtering-the-logger-output)
> 4. Copy the logger via 📋 [export/copy button at the top right](https://github.com/gorhill/uBlock/wiki/The-logger#export-dialog)
> 5. Paste the logs to https://logpasta.com and share the link here

sorry for the late reply but I was waiting for this week's free games to try this.

I tried this just now and the error didn't occur.

I then tried this on the new Battlefield F2P game, the error occurred again "There was a problem checking out your order please try again later". I filtered by "blocked", and there was literally nothing, the logger was empty.

Then I tried again on the BF game and the error didn't occur...

But like I said, if I whitelist the tracking domain then the error never occurs...

I will try this again later and also for next week's free games.

**Comment by stephenhawk8054** [comment]
Did you open the logger before reproducing the issue?

**Comment by muchqs** [comment]
yes I did. I opened the logger before I opened the game's page. I tried to record the process on the next try but the elusive error didn't occur...

**Comment by krystian3w** [comment]
Remember to clear your browser cache in epicgames domain, otherwise the page cache may be considered valid and bypassed by uBO's blocking.
